.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run lint -- --fix
```

Comment:
`--fix` is already defined in the package.json for the scripts `lint` so it can be omitted here

Reply:
Sure, anything else?

Reply:
LGTM